examples/demo_controlling_integration.m

```diff
@@ -21,13 +21,13 @@
 x.AB.add('Leak','gbar',.099,'E',-50);
 
 
-x.AB.NaV.add('IntegralController','tau_m',666);
-x.AB.CaT.add('IntegralController','tau_m',55555);
-x.AB.CaS.add('IntegralController','tau_m',45454);
-x.AB.ACurrent.add('IntegralController','tau_m',5000);
-x.AB.KCa.add('IntegralController','tau_m',1250);
-x.AB.Kd.add('IntegralController','tau_m',2000);
-x.AB.HCurrent.add('IntegralController','tau_m',125000);
+x.AB.NaV.add('oleary/IntegralController','tau_m',666);
```

Comment:
No, there's no explicit need for this. I moved `IntegralController.hpp` into a new folder called `oleary`, but since it is the only mechanism with the name `IntegralController` it is unnecessary to specify, at least, at this time.